samples/drivers/video/capture_to_lvgl/boards/mini_stm32h743.overlay

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2024 Charles Dias <charlesdias.cd@outlook.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+&pll {
+	div-m = <5>;
+	mul-n = <96>;
+	div-p = <2>;
+	div-q = <2>;
+	div-r = <2>;
+	clocks = <&clk_hse>;
+	status = "okay";
+};
+
+&rcc {
```

Comment:
Non blocking: If you're changing the clock for some specific reason, let's put it as a comment.

Reply:
I'll remove this configuration because it's not necessary to use the "VIDEO_SW_GENERATOR".